src/tests/ftest/erasurecode/ec_aggregation.yaml

```diff
@@ -0,0 +1,72 @@
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+    - server-C
+    - server-D
+    - server-E
+  test_clients:
+    - client-A
+    - client-B
+    - client-C
+timeout: 7200
+server_config:
+  engines_per_host: 2
+  name: daos_server
+  servers:
+    0:
+      pinned_numa_node: 0
+      nr_xs_helpers: 1
+      fabric_iface: ib0
+      fabric_iface_port: 31317
+      log_file: daos_server0.log
+      bdev_class: nvme
+      bdev_list: ["aaaa:aa:aa.a"]
+      scm_class: dcpm
+      scm_list: ["/dev/pmem0"]
+      scm_mount: /mnt/daos0
+      log_mask: ERR
+    1:
+      pinned_numa_node: 1
+      nr_xs_helpers: 1
+      fabric_iface: ib1
+      fabric_iface_port: 31417
+      log_file: daos_server1.log
+      bdev_class: nvme
+      bdev_list: ["bbbb:bb:bb.b"]
+      scm_class: dcpm
+      scm_list: ["/dev/pmem1"]
+      scm_mount: /mnt/daos1
+      log_mask: ERR
+pool:
+    mode: 146
+    name: daos_server
+    scm_size: 200G
+    nvme_size: 500G
+    svcn: 1
+    control_method: dmg
+    pool_query_timeout: 30
+container:
+    type: POSIX
+    control_method: daos
+ior:
+  api: "DFS"
+  client_processes:
+   np: 48
+  dfs_destroy: False
+  iorflags:
+   flags: "-w -W -F -k -G 1 -vv"
+   read_flags: "-r -R -F -k -G 1 -vv"
+  test_file: /testFile
+  repetitions: 1
+  chunk_block_transfer_sizes:
+   #[ChunkSize, BlocksSize, TransferSize]
+   - [32M, 64M, 2048]
+   - [32M, 16M, 128]
```

Comment:
The transfer size here will be less than 4K in both the scenarios. Isn't it, that there will be no data going to NVMe at all in these cases, hence aggregation check is of no use, if I understand it right.
Might want to increase transfer size to 4K or greater.

Reply:
So I am not sure what is mean by aggregation check no use?
Yes, data will not go to NVMe at first but once Aggregation starts data will be moved from SCM to NVMe. So we check when SCM size starting increasing as an indication that aggregation started.